geoprocessing/src/migrations/geoprocessing/1611828857842-addFeatureScenarioOutputEntity.ts

```diff
@@ -0,0 +1,32 @@
+import { Logger } from '@nestjs/common';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { PostgreSQLUtils } from 'src/utils/postgresql.utils';
+
+export class addFeatureScenarioOutputEntity1611828857842
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Only CREATEDB privilege required in 13+ rather than SUPERUSER (ht @agnessa)
+    if (await PostgreSQLUtils.version13Plus()) {
```

Comment:
we don't need this wrapper here as we don't run queries that need privileges this connection wouldn't have by default - no harm in keeping it, but I think it may be confusing.